packages/node/tsconfig.json

```diff
@@ -8,10 +8,12 @@
     "lib": ["es2019"],
     "outDir": "./dist",
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
unfortunately this flag is required since the `api` core doesn't play nicely with TS strict mode (or maybe it's because we support Node 14 in this repo? not sure). i started https://github.com/readmeio/api/pull/696 which could maybe help?

![CleanShot 2023-08-23 at 12 35 13@2x](https://github.com/readmeio/metrics-sdks/assets/8854718/2e3aacc8-5cb7-4c42-9af3-efd26ed9c893)